### PR TITLE
Add multiple tables.

### PR DESCRIPTION
**Pull request by Nowembabazi**
**##In this milestone i achieved the following;**
-Add multiple tables.
-Use primary key & foreign key mechanism for joining tables.
-Query multiple tables.
-Prepare complex queries that answer analytical questions.

**##BELOW ARE THE SCREENSHOOTS;**
image
image
image
image
image
image
image

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -31,5 +31,6 @@ VALUES ('Boarmon', '06-07-2005', 7, true, 20.4),
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
 VALUES ('Boarmon', '06-07-2005', 7, true, 20.4), 
 
+
```
Comment (Davidosky007) [inline data.sql:34]: * Kindly Insert the following data into the owners table:

> Sam Smith 34 years old.
> Jennifer Orwell 19 years old.
> Bob 45 years old.
> Melody Pond 77 years old.
> Dean Winchester 14 years old.
> Jodie Whittaker 38 years old
* And also [these requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md#:~:text=Insert%20the%20following,data.sql%20file.) are missing kindly insert all the necessary datas

File: queries.sql [hunk 2]
```diff
@@ -95,4 +95,22 @@ SELECT neutered, count(neutered) as result FROM animals GROUP BY neutered ORDER
 SELECT species, MAX(weight_kg) AS max_weight, MIN(weight_kg) AS min_weight FROM animals GROUP BY species;
 
 /* What is the average number of escape attempts per animal type of those born between 1990 and 2000? */
-SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
\ No newline at end of file
+SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
+
+SELECT animals.name, species.name FROM animals JOIN species ON animals.species_id = species.id WHERE species.name = 'Pokemon';
```
Comment (Davidosky007) [inline queries.sql:100]: * Kindly add this missing query `What animals belong to Melody Pond?` [Check requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md#:~:text=What%20animals%20belong%20to%20Melody%20Pond%3F)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @Nowembabazi 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights
* Well written PR description 👍 
* Proper use of Giflow 👍 
* Good looking readme file 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by m05e5 — APPROVED** [review]
Hi @Nowembabazi 

You did great. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
CleverScornfulBuffalo-size_restricted

## Highlights
✔️ Descriptive PR 

## Optional suggestions


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**m05e5**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**